src/components/MyProfileActivityList/index.js

```diff
@@ -278,7 +292,63 @@ class MyProfileActivityList extends Component {
       );
     }
 
-    return <div>{myProfileInvolvementsList}</div>;
+    return (
+      <div>
+        {myProfileInvolvementsList}
+        <Snackbar
```

Comment:
Adds a Snackbar component for each activity's privacy switch. It's shown here instead of inside of "myProfileInvolvementsList" because the variable "myProfileInvolvementsList" has a conditional which determines what content is returned. Therefore, having the snackbar inside of "myProfileInvolvementsList" would require the Snackbar component to be defined two times which is duplicated code.